src/Nelmio/Alice/Fixtures/Builder/Methods/ReferenceRangeName.php

```diff
@@ -0,0 +1,70 @@
+<?php
+
+namespace Nelmio\Alice\Fixtures\Builder\Methods;
+
+use Nelmio\Alice\Fixtures\Fixture;
+use Nelmio\Alice\Instances\Collection;
+
+final class ReferenceRangeName implements MethodInterface
+{
+    /**
+     * @var array $matches
+     */
+    private $matches = [];
+
+    /**
+     * @var \Nelmio\Alice\Instances\Collection
+     */
+    private $objects;
+
+    /**
+     * ReferenceRangeName constructor.
+     * @param \Nelmio\Alice\Instances\Collection $objects
+     */
+    public function __construct(Collection $objects)
+    {
+        $this->objects = $objects;
+    }
+
+    /**
+     * {@inheritDoc}
+     */
+    public function canBuild($name)
+    {
+        return 1 === preg_match('#\{@([0-9a-zA-Z\._\-]+)(\*?+)\}#i', $name, $this->matches);
+    }
+
+    /**
+     * {@inheritDoc}
+     */
+    public function build($class, $name, array $spec)
+    {
+        $fixtures = [];
+
+        $referenceName = $this->matches[1];
+        $referenceAll = $this->matches[2] == '*';
+
+        if($referenceAll) {
+            $keys = $this->objects->getKeysByMask($referenceName.".+");
+            foreach($keys as $currentIndex => $key) {
+                $instance = $this->objects->find($key);
+                $currentName = str_replace($this->matches[0], $key, $name);
+                $fixture = new Fixture($class, $currentName, $spec, $instance);
```

Comment:
add space between L52-53: maybe it's a personal preference but I find it clearer that way